packages/data/src/world/mq/dodongo_cavern_mq.yml

```diff
@@ -2,159 +2,279 @@
   dungeon: DC
   exits:
     "Death Mountain": "true"
-    "Dodongo Cavern Main": "has_bombflowers || can_hammer || has_blue_fire_arrows_mudwall"
-"Dodongo Cavern Main":
+    "Dodongo Cavern Main Room Lower": "has_bombflowers || can_hammer || has_blue_fire || has_blue_fire_arrows_mudwall"
+"Dodongo Cavern Main Room Lower":
   dungeon: DC
   exits:
-    "Dodongo Cavern": "true"
-    "Dodongo Cavern Skull": "has_explosives"
-    "Dodongo Cavern Staircase": "has_bombflowers || can_hammer"
-    "Dodongo Cavern Upper Ledges": "has_explosives_or_hammer || can_use_din"
-    "Dodongo Cavern Lower Tunnel": "has_explosives_or_hammer || (event(DC_MQ_SHORTCUT) && has_goron_bracelet)"
-    "Dodongo Cavern Bomb Bag Ledge": "is_adult"
+    "Dodongo Cavern": "has_explosives_or_hammer || has_blue_fire || has_blue_fire_arrows_mudwall" #We need to see if you can go backwards without breaking it, like other mudwalls.
```

Comment:
You actually can by using the front platform, have to throw it before it explodes. but this may become moot anyway if you can walk through it. Which should be testable on the OOTR branches that have boss mixed pools right?

Reply:
I tried, but it didn't look like it reached. Were you adult or child?

Reply:
adult. i can try as child

Reply:
yeah adult can do it easily. child doesnt appear to be fast enough to get even the throw close enough.

Reply:
Added adult with bombflowers